src/lib/pay-request/services/connector/types.ts

```diff
@@ -26,6 +26,7 @@ export interface Charge {
 }
 
 export interface GatewayAccount {
+  // this may be inconsistent in being returned as either or a string or number
   gateway_account_id: string;
```

Comment:
should this be a union type then?

```ts
interface GatewayAccount {
    gateway_account_id:string|number;
    ...
}
```

Reply:
i mean probably but a lot of other stuff complains then